RevenueCatUI/CustomerCenter/Views/UIKit Compatibility/CustomerCenterViewController.swift

```diff
@@ -33,23 +33,176 @@ import SwiftUI
 @available(watchOS, unavailable)
 public class CustomerCenterViewController: UIHostingController<CustomerCenterView> {
 
+    // Handler for when the view controller is dismissed
+    public var onDismiss: (() -> Void)?
+
+    // Stored handlers for later updating if needed
+    private var restoreStartedHandler: CustomerCenterView.RestoreStartedHandler?
+    private var restoreCompletedHandler: CustomerCenterView.RestoreCompletedHandler?
+    private var restoreFailedHandler: CustomerCenterView.RestoreFailedHandler?
+    private var showingManageSubscriptionsHandler: CustomerCenterView.ShowingManageSubscriptionsHandler?
+    private var refundRequestStartedHandler: CustomerCenterView.RefundRequestStartedHandler?
+    private var refundRequestCompletedHandler: CustomerCenterView.RefundRequestCompletedHandler?
+    private var feedbackSurveyCompletedHandler: CustomerCenterView.FeedbackSurveyCompletedHandler?
+
+    // Legacy handler for backward compatibility
+    private var legacyHandler: DeprecatedCustomerCenterActionHandler?
+
+    // MARK: - Initialization
+
     /// Create a view controller to handle common customer support tasks
     /// - Parameters:
     ///   - customerCenterActionHandler: An optional `CustomerCenterActionHandler` to handle actions
     ///   from the Customer Center.
+    @available(*, deprecated, message: "Use the initializer with individual action handlers instead")
     public init(
         customerCenterActionHandler: CustomerCenterActionHandler? = nil
```

Comment:
A similar comment here.
```suggestion
        customerCenterActionHandler: CustomerCenterActionHandler?
```

To disambiguate the `init`s with all parameters set to its default values (`init()` in this case), I would remove the default parameter value of the `customerCenterActionHandler` so that 
1. Only customers explicitly using `init(customerCenterActionHandler: ...)` would get the deprecation warning
2. `init()` is not ambiguous.